content/collections/account-management/en/self-service-data-deletion-in-amplitude.md

```diff
@@ -0,0 +1,45 @@
+---
+id: 1b48f869-0e38-45ac-a6d5-03184a2ac77d
+blueprint: account-management
+title: 'Self-service data deletion in Amplitude'
+this_article_will_help_you:
+  - 'Delete data from Amplitude permanently'
+landing: false
+exclude_from_sitemap: false
+updated_by: 5817a4fa-a771-417a-aa94-a0b1e7f55eae
+updated_at: 1728428160
+---
+Sometimes, you may need to permanently remove data from your Amplitude projects in a timely manner. For example, maybe your product sent incorrect data to Amplitude last month, but that data has since been corrected, and you’d like to remove the original, incorrect events or properties.
+
+Amplitude’s **self-service data deletion** feature lets you do this, without involving Amplitude personnel. Simply specify which events or properties you want to delete and send your deletion request to the Amplitude deletion queue.
+
+## Before you begin
+
+This feature is available to users on Growth and Enterprise plans only. Growth users get one deletion task, to include up to one billion events scanned, per month. Enterprise users get 10 tasks, each of which can include as many as one billion events scanned, per month.
+
+Only users with Admin permissions can use this feature.
+
+**You can’t cancel your request** to delete data once your submitted request is in the Amplitude deletion queue.
+
+## Submit a data deletion task
+
+To delete your Amplitude data, you must create a **task**. You can request to delete events or properties. When you delete events, Amplitude will also delete all properties and property data associated with those events. When deleting properties, specify whether you want to limit deletion to properties attached to specific events, or to delete these properties for all events.
```

Comment:
```suggestion
To delete your Amplitude data, create a **task**. Self-service data deletion supports both events and properties. When you delete events, Amplitude deletes all properties and property data associated with those events. When you delete properties, specify whether you want to limit deletion to properties attached to specific events, or to delete these properties for all events.
```
What are your thoughts on "You must" and "you can"?

Also, most of this paragraph doesn't relate to tasks, but seem to describe self-service deletion itself. Maybe move it up a bit?

Reply:
My sense is that they both have their place, but it's not nearly as prominent as one might assume. It's probably not needed here.

Reply:
I put this here (as opposed to earlier) because I thought it was too detailed for the intro, but was needed for context prior to jumping into the procedure itself. I'll separate the two concepts, but keep the material closer to the procedure.